build/travis/job2_AppImage/bintray.sh

```diff
@@ -40,20 +40,21 @@ if [ $(env | grep TRAVIS_PULL_REQUEST ) ] ; then
   fi
 fi
 
-BINTRAY_USER=$BINTRAY_USER # env
-BINTRAY_API_KEY=$BINTRAY_API_KEY # env
+BINTRAY_USER="${BINTRAY_USER:?Environment variable missing/empty!}" # env
+BINTRAY_API_KEY="${BINTRAY_API_KEY:?Environment variable missing/empty!}" # env
 
-BINTRAY_REPO_OWNER=$BINTRAY_REPO_OWNER #env
-[ "$BINTRAY_REPO_OWNER" ] || BINTRAY_REPO_OWNER="musescore"
+BINTRAY_REPO_OWNER="${BINTRAY_REPO_OWNER:-$BINTRAY_USER}" # env, or use BINTRAY_USER as default
+[ "${BINTRAY_REPO_OWNER}" == "musescore" ] && TRUSTED="true" || TRUSTED="false"
```

Comment:
This swaps the logic for BINTRAY_REPO_OWNER. Now @lasconic will have to set `BINTRAY_REPO_OWNER=musescore` in Travis, but for everyone else it will default to BINTRAY_USER. This was probably what @ericfont wanted all along.